websockets/commands.go

```diff
@@ -136,7 +136,18 @@ type SubmitResult struct {
 	EngineResultCode    int                    `json:"engine_result_code"`
 	EngineResultMessage string                 `json:"engine_result_message"`
 	TxBlob              string                 `json:"tx_blob"`
-	Tx                  interface{}            `json:"tx_json"`
+	Tx                  struct {
```

Comment:
```suggestion
	Tx                  struct {
```

SubmitResultTx struct 따로 만드는게 어떤가여?